spec_11/index.js

```diff
@@ -0,0 +1,13 @@
+
+const findFactorial = ( n ) => {
+  let factorial = 1
+  for ( let i = 1; i <= n; i++ ) {
```

Comment:
Consider adding whitespace for clarity.  I try to divide functions into the setup, body, and return sections with the use of whitespace.